Eatery Blue/UI/SettingsSupport/SettingsSupportView.swift

```diff
@@ -95,11 +95,10 @@ struct SettingsSupportView: View {
                             .font(Font(UIFont.preferredFont(for: .body, weight: .semibold)))
                         Spacer()
                     }
-                }
+                })
                 .foregroundColor(.white)
                 .background(Color("EateryBlue"))
                 .clipShape(Capsule())
-                .buttonStyle(.plain)
```

Comment:
Nice! Tapping on the empty space of a`Button` with a `.plain` style won't trigger the action. Sorry, I know you asked me to help about this but I was a bit busy with iOS course stuff. Good job figuring this out!